lib/wallet-account.js

```diff
@@ -164,6 +164,8 @@ class WalletConnection {
         currentUrl.searchParams.delete('public_key');
         currentUrl.searchParams.delete('all_keys');
         currentUrl.searchParams.delete('account_id');
+        currentUrl.searchParams.delete('meta');
```

Comment:
Why do we need these changes here?

Reply:
I don't know . Whenever we run `yarn build` these changes are automatically made to `lib/wallet-account.js`